installer/conf/omsagent.d/syslog.conf

```diff
@@ -0,0 +1,12 @@
+<source>
+  type syslog
+  port %PORT%
```

Comment:
Since this port and the port replaced in monitor.conf are two different types of ports (syslog vs monitor agent) could they be templated with different strings?